src/middlewared/middlewared/plugins/iscsi.py

```diff
@@ -571,8 +469,8 @@ class iSCSITargetExtentModel(sa.Model):
     iscsi_target_extent_xen = sa.Column(sa.Boolean(), default=False)
     iscsi_target_extent_rpm = sa.Column(sa.String(20), default='SSD')
     iscsi_target_extent_ro = sa.Column(sa.Boolean(), default=False)
-    iscsi_target_extent_legacy = sa.Column(sa.Boolean(), default=False)
     iscsi_target_extent_enabled = sa.Column(sa.Boolean(), default=True)
+    iscsi_target_extent_vendor = sa.Column(sa.Text(), default='TrueNAS')
```

Comment:
I've migrated those `default=` from Django because I did not have time back then to figure out what defaults were not set by middleware plugins causing NULL errors. For new fields we should always have defaults in plugin and not rely on model in setting them.